api-common/src/main/kotlin/io/qalipsis/api/meters/MeterSnapShotImpl.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 AERIS IT Solutions GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package io.qalipsis.api.meters
+
+import java.time.Instant
+
+/**
+ * Implementation of [MeterSnapshot] to store meter measurement.
+ *
+ * @author Francisca Eze
+ */
+class MeterSnapShotImpl<T : Meter<T>>(
+    override val timeStamp: Instant,
```

Comment:
```suggestion
    override val timestamp: Instant,
```

Reply:
The implementation should not be the API, but in the OSS only.

Reply:
@ericjesse what about its usage in the plugins where I have to test that the plugins publish these snapshots. If the implementation is on the OSS alone then I wouldn't have access to them within the plugins.

Reply:
You would still have access to the interface of it. Isn't it enough?

Reply:
No I need to create instances of MeterSnapshotsImpl that we assumed have been exported by the oss

Reply:
But you can also use mocks instead of concrete implementations. Or you can create an implementation in the testFixtures of the api-common module, that is used for testing only if you need to simplify the creation of the snapshots.

Reply:
Okay I will use mocks for it